pop_tools/xgcm_util.py

```diff
@@ -98,6 +98,13 @@ def relabel_pop_dims(ds):
             else:
                 dims = new_spatial_dims
             ds_new[vname] = xr.Variable(dims, da.data, da.attrs, da.encoding, fastpath=True)
+    old_coords = ['nlat', 'nlon']
+    for coord in old_coords:
+        if coord in ds_new.coords:
+            ds_new = ds_new.drop_vars(coord)
+        assert coord not in ds_new.dims
+    if 'z_w_top' and 'z_w' in ds_new.dims:
```

Comment:
if `'z_w_top' and 'z_w' in ds_new.dims` --> `if 'z_w_top' in ds_new.dims and 'z_w' in ds_new.dims`: